composer.json

```diff
@@ -15,5 +15,6 @@
     "psr-4": {
       "Dorgflow\\": ""
     }
-  }
+  },
+  "bin": ["bin/dorgflow"]
```

Comment:
Can't we just leave the dorgflow file where it is, and give a different value to composer in the 'bin' property? Looking at how the bin/ part is given here, it doesn't look like the bin/ folder is magic in any way.